samples/benchmarks/coremark/boards/nrf52833dk_nrf52833.conf

```diff
@@ -0,0 +1,6 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+CONFIG_LOG_MODE_MINIMAL=y
```

Comment:
Moving CONFIG_LOG_MODE_MINIMAL from global to board config seams a bit confusing. Consider changing commits order :)

Reply:
Which commits?

Reply:
I need to keep this order as the IMMEDIATE logging does not fit into the memory allocated for Pepper. That's why commits are in the current order.